documentation/README.html

```diff
@@ -0,0 +1,27 @@
+<h1> CIS compliance package framework for Unix </h1>

```

Comment:
I put this in README.org, github renders it automagically and then it also reads a bit nicer for people that live on a console. Do you have any objection to sticking to README.org?